src/app/search/search-workflow-table/search-workflow-table.component.ts

```diff
@@ -21,16 +21,17 @@ import { DockstoreService } from '../../shared/dockstore.service';
 import { Workflow } from '../../shared/swagger';
 import { SearchEntryTable } from '../search-entry-table';
 import { SearchQuery } from '../state/search.query';
-
+import {SearchService} from "../state/search.service";
+//this component refers to search page not workflow listing search
 @Component({
   selector: 'app-search-workflow-table',
   templateUrl: './search-workflow-table.component.html',
   styleUrls: ['./search-workflow-table.component.scss']
 })
 export class SearchWorkflowTableComponent extends SearchEntryTable implements OnInit {
   public dataSource: MatTableDataSource<Workflow>;
-  constructor(private dockstoreService: DockstoreService, protected dateService: DateService, private searchQuery: SearchQuery) {
-    super(dateService);
+  constructor(private dockstoreService: DockstoreService, protected dateService: DateService,  protected searchQuery: SearchQuery, protected searchService: SearchService) {
```

Comment:
I don't think you need the `protected` modifier for the new parameters you added. You are just passing them along to the super, which already creates them as properties.